Fix wrong bin directory for Symfony2 applications

yhoiseth opened the pull request:
Because Symfony has set `config.bin-dir = "bin"` in `composer.json`, binaries are installed to `bin` rather than `vendor/bin`.

I did the following:

1. Installed Symfony 2.8 using the Symfony installer: `symfony new behat-symfony-tutorial 2.8`.
2. Installed Behat: `cd behat-symfony-tutorial && composer require --dev behat/behat
3. Tried running Behat: `vendor/bin/behat`

## Expected result
```
[Behat\Behat\Context\Exception\ContextNotFoundException]       
  `FeatureContext` context class not found and can not be used.
```

## Actual result
`zsh: no such file or directory: vendor/bin/behat`

To fix this, I used `bin/behat` instead.

I think the documentation should be updated to reflect Symfony 2.8 standard behavior.

stof commented:
I would rather just add a note there, and keep the example unchanged. Symfony projects don't always overwrite the bin dir (and btw, new projects created on Symfony 3 don't do it anymore but keep using the composer default setting)

yhoiseth commented:
@stof 

> I would rather just add a note there, and keep the example unchanged. Symfony projects don't always overwrite the bin dir (and btw, new projects created on Symfony 3 don't do it anymore but keep using the composer default setting)

Good point. I've reverted the commit and added a tip instead.

Also, I added instructions for initialization because the rest of the tutorial assumes that Behat has been initialized.
